service/gateway/router.go

```diff
@@ -95,5 +96,11 @@ func (g *Gateway) registerHandler(r *mux.Router) {
 		Name(replicateObjectPieceRouterName).
 		Methods(http.MethodPut).
 		HandlerFunc(g.replicatePieceHandler)
+	r.Path(model.UniversalENdpointPath).
+		Name(getObjectByUniversalEndpointName).
+		Methods(http.MethodGet).
+		HandlerFunc(g.getObjectByUniversalEndpointHandler)
+	//redirect for universal endpoint
```

Comment:
Should we insert the comment line between 98 and 99?

Reply:
http.Handle("/", r) is the redirect router (currently only used by universal endpoint, but can be used by other features in the future too), line between 98 and 99 is the router for universal endpoint. Just added a new comment at line between 98 and 99.

Reply:
Refine `//commentary` to `// commentary` , add a space before the comment.

Reply:
If you have time, you can supplement the unit test in router_test.go to prevent others from destroying this part of the logic in the future.

Reply:
Thanks for pointing out! Will update to // commentary in next pr